metricflow/test/plan_conversion/test_dataflow_to_sql_plan.py

```diff
@@ -1126,3 +1127,111 @@ def test_dimension_with_joined_where_constraint(
         sql_client=sql_client,
         node=dataflow_plan.sink_output_nodes[0].parent_node,
     )
+
+
+@pytest.mark.sql_engine_snapshot
+def test_metric_time_only(
+    request: FixtureRequest,
+    mf_test_session_state: MetricFlowTestSessionState,
+    dataflow_plan_builder: DataflowPlanBuilder,
+    dataflow_to_sql_converter: DataflowToSqlQueryPlanConverter,
+    sql_client: SqlClient,
+) -> None:
+    """Tests querying only metric time."""
```

Comment:
nit: do you mind moving these to a new module in the query_rendering directory? I know it's a little annoying but these are really query rendering tests, and I'd like to keep this file from becoming the accumulator of all the SQL snapshot tests again.